### PR TITLE
Bug fix: Remove duplicate https being present in response from POST endpoint

### PR DESCRIPTION
**Pull request by Koziad**
When making a POST request to the `c/{url}` endpoint and the URL path parameter already contained http/-s in it, then the redirect from shortened URL wouldn't work. Example:

1. POST request being with the following url: `/c/https://www.foo.bar`
2. Response body would return: 
```json
{
    "expires": "2022-08-25T08:37:19.134329099+02:00",
    "uri": "https://https://www.foo.bar", // <-- problem is here
    "key": "B2H7X-0SdYwG",
    "gurl": "http://localhost:9999/b/B2H7X-0SdYwG",
    "token": "291ecd5e-dbe9-41be-b5d2-9e26ff0116fa"
}
```
3. GET request to `http://localhost:9999/b/B2H7X-0SdYwG` will then not work.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -182,6 +183,10 @@ func main() {
 				return err
 			}
 
+			// Remove http(s) prefix if provided to avoid duplication
```
Comment (PatchesPrime) [inline main.go:186]: Can you think of a way to do it without regexp? :thinking:

File: main.go [hunk 2]
```diff
@@ -182,6 +183,10 @@ func main() {
...
+			re := regexp.MustCompile("^(http|https)://")
+			uri = re.ReplaceAllString(uri, "")
+
```
Comment (PatchesPrime) [inline main.go:198]: Could you do this without regex, though? :thinking:

**Review by PatchesPrime — CHANGES_REQUESTED** [review]
Regexp are pretty slow, and this operation could easily be done without it.
Good find, lets try it without a dependency and regexp :+1:

File: main.go [hunk 3]
```diff
@@ -184,8 +183,12 @@ func main() {
 			}
 
 			// Remove http(s) prefix if provided to avoid duplication
-			re := regexp.MustCompile("^(http|https)://")
-			uri = re.ReplaceAllString(uri, "")
+			for i, rune := range uri {
+				if rune == '/' {
```
Comment (PatchesPrime) [inline main.go]: Shouldn't this still not work? It would, in my thinking, trigger on the first slash and include the second in the slice.

**Review by PatchesPrime — CHANGES_REQUESTED** [review]

File: main.go (hunk 3 above)
Comment (Koziad) [reply main.go]: It would indeed trigger on the first slash, but the redirect works when using the provided `gurl` field from the response body

**Review by PatchesPrime — APPROVED** [review]
Sorry it took so long to get back, but looks good to me.

**Pull request merged**